src/ecAst.mli

```diff
@@ -111,7 +112,7 @@ and instr_node =
   | Sif       of expr * stmt * stmt
   | Swhile    of expr * stmt
   | Smatch    of expr * ((EcIdent.t * ty) list * stmt) list
-  | Sassert   of expr
+  | Sraise    of EcPath.path
```

Comment:
It will be better to allow expression here. 
It will allow to have exception with arguments